pkg/manifests/ext_proc.yaml

```diff
@@ -71,7 +71,7 @@ spec:
     spec:
       containers:
       - name: inference-gateway-ext-proc
-        image: us-central1-docker.pkg.dev/k8s-staging-images/gateway-api-inference-extension/epp:main
+        image: gcr.io/jeffluoo-gke-dev/epp:latest
```

Comment:
you'll want to remove this before submitting :)

Reply:
Done.